bodhi/server/consumers/masher.py

```diff
@@ -902,6 +903,36 @@ def compose_atomic_trees(self):
         mashed_repos = dict([('-'.join(os.path.basename(repo).split('-')[:-1]), repo)
                              for repo in self.state['completed_repos']])
         for tag, mash_path in mashed_repos.items():
+            #  TODO: Find out the actual values for the tag
+            #  also for the code below, should we use the mash_path instead
+            #  of current_workdir?
+            #  * Pungi config repos hsa $arch, does it matter in bodhi?
```

Comment:
Does Bodhi create a repo for multiple architectures? I think it does. Using `$arch` in the URL then allows to have the same config for multiple arches.